targets/csharp/source/PlayFabSDK/source/PlayFabSDK.csproj.ejs

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard1.1;net45;portable45-net45+win8+wpa81</TargetFrameworks>
```

Comment:
Same as above. Are these changes intentional?

Reply:
portable45 had an issue, and so did the win8 stuff. I put back netstandard 2.0 and 1.1 in this project file and it still ran fine without issues so I can add those back (netcore seems to be what is end of life, standard seems to still be around)